src/input.cpp

```diff
@@ -686,6 +710,16 @@ class deinit_t : public platf::deinit_t {
 [[nodiscard]] std::unique_ptr<platf::deinit_t> init() {
   platf_input = platf::input();
 
+  if(config::input.keep_first_gamepad) {
+    //Create a dummy first gamepad that does not disconnect across sessions
```

Comment:
We should be connecting our dummy gamepads as part of the `launch` and `resume` processing according to the gamepad mask specified by the client in the `remoteControllersBitmap` parameter.